src/main/java/core/basesyntax/service/FruitList.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitTransaction;
+import java.util.List;
+
+public interface FruitList {
+    List<FruitTransaction> getFruitList(List<FruitTransaction> list);
```

Comment:
The interface name 'FruitList' is a bit misleading as it does not represent a list but rather an operation that can be performed on a list. Consider renaming it to reflect its purpose, such as 'FruitTransactionProcessor' or similar to indicate it processes a list of 'FruitTransaction' objects.